flink-cdc-runtime/src/main/java/com/ververica/cdc/runtime/parser/validate/FlinkCDCOperatorTable.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.runtime.parser.validate;
+
+import com.ververica.cdc.runtime.functions.BuiltInScalarFunction;
+import com.ververica.cdc.runtime.functions.BuiltInTimestampFunction;
+import com.ververica.cdc.runtime.functions.FlinkCDCReturnTypes;
+import org.apache.calcite.sql.SqlFunction;
+import org.apache.calcite.sql.SqlFunctionCategory;
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlKind;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.SqlSyntax;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+import org.apache.calcite.sql.type.OperandTypes;
+import org.apache.calcite.sql.type.ReturnTypes;
+import org.apache.calcite.sql.type.SqlOperandCountRanges;
+import org.apache.calcite.sql.type.SqlTypeFamily;
+import org.apache.calcite.sql.type.SqlTypeName;
+import org.apache.calcite.sql.type.SqlTypeTransforms;
+import org.apache.calcite.sql.util.ReflectiveSqlOperatorTable;
+import org.apache.calcite.sql.validate.SqlNameMatcher;
+import org.apache.calcite.sql.validate.SqlNameMatchers;
+
+import javax.annotation.Nullable;
+
+import java.util.List;
+
+/** FlinkCDCOperatorTable to generate the metadata of calcite. */
+public class FlinkCDCOperatorTable extends ReflectiveSqlOperatorTable {
```

Comment:
Could we remove all `FlinkCDC` prefix as we can limit these classes'  usage to transform component inside?

Reply:
`Sql` should be kept here as this is a table used to lookup various `SqlOperator` like `SqlFunction`, how about `TransformSqlOperatorTable `